packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1083,7 +1086,10 @@ export class GarbageCollector implements IGarbageCollector {
 	 * before runnint GC next.
 	 */
 	private triggerAutoRecovery(nodePath: string) {
-		if (this.mc.config.getBoolean(disableAutoRecoveryKey) === true) {
+		if (
+			!this.configs.tombstoneAutorecoveryEnabled ||
```

Comment:
Maybe we should just use sweepEnabled here instead of adding another config?

Reply:
I started that way but I like it being more explicit.  It's not obvious IMO that they would be configured together.

Reply:
That said, if you do a config clean up PR and combine them, I'll probably be ok with that, depending on how it's stated (names and documentation)